storybook/qmlTests/tests/tst_TokenSelectorViewAdaptor.qml

```diff
@@ -0,0 +1,114 @@
+import QtQuick 2.15
+import QtTest 1.15
+
+import Models 1.0
+
+import StatusQ.Core.Utils 0.1
+
+import AppLayouts.Wallet.stores 1.0
+import AppLayouts.Wallet.adaptors 1.0
+
+import shared.stores 1.0
+
+Item {
+    id: root
+    width: 600
+    height: 400
+
+    QtObject {
+        id: d
+
+        readonly property var flatNetworks: NetworksModel.flatNetworks
+        readonly property var currencyStore: CurrenciesStore {}
+        readonly property var assetsStore: WalletAssetsStore {
+            id: thisWalletAssetStore
+            walletTokensStore: TokensStore {
+                plainTokensBySymbolModel: TokensBySymbolModel {}
+            }
+            readonly property var baseGroupedAccountAssetModel: GroupedAccountsAssetsModel {}
+            assetsWithFilteredBalances: thisWalletAssetStore.groupedAccountsAssetsModel
+        }
+    }
+
+    Component {
+        id: componentUnderTest
+        TokenSelectorViewAdaptor {
+            assetsModel: d.assetsStore.groupedAccountAssetsModel
+            flatNetworksModel: d.flatNetworks
+            currentCurrency: d.currencyStore.currentCurrency
+        }
+    }
```

Comment:
Similarly here, what is needed are simple models for `assetsModel` and `flatNetworksModel`. No need for stores in unit test of adaptor.